### PR TITLE
VolSync: Install via OLM

### PR DESCRIPTION
**Pull request by JohnStrunk**
Install the VolSync operator via OLM bundle instead of Helm

### CODE REVIEW

**Comment by JohnStrunk** [comment]
/hold
WIP

**Comment by JohnStrunk** [comment]
Waiting for backube/volsync#314

**Comment by JohnStrunk** [comment]
/unhold
/cc @tesshuflower
This switches our OpenShift CI runs to generate a bundle and install using OLM.

**Review by tesshuflower — APPROVED** [review]
/lgtm

File: ci-operator/config/backube/volsync/backube-volsync-main.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.18-openshift-4.12
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (JohnStrunk) [inline ci-operator/config/backube/volsync/backube-volsync-main.yaml:13]: Builds from the 4.10 index since that's the latest released index

**Review by tesshuflower — APPROVED** [review]
/lgtm

**Pull request merged**